sdk/communication/Azure.Communication.Chat/README.md

```diff
@@ -229,9 +226,9 @@ Use `SendMessage` to send a message to a thread.
 
 ```C# Snippet:Azure_Communication_Chat_Tests_Samples_SendMessage
 var content = "hello world";
-var priority = ChatMessagePriority.Normal;
+var type = ChatMessageType.Html;
```

Comment:
If we want this to be the simple example, I would say let's not even mention type specially not Html